omero/sysadmins/windows/server-installation.txt

```diff
@@ -328,120 +278,35 @@ is having a properly configured database.
 
 -  Create a non-superuser database user (make sure to note down the name
    and password) using pgAdmin III. You can find pgAdmin III on
-   the Start Menu under :menuselection:`Programs --> PostgreSQL 9.1 --> pgAdmin III`:
+   the Start Menu under :menuselection:`Programs --> PostgreSQL --> pgAdmin III`:
 
-   #. Double-click on the :guilabel:`PostgreSQL 9.1` database (or right-
+   #. Double-click on the :guilabel:`PostgreSQL` database (or right-
       click and choose :guilabel:`Connect`) and provide your *postgres* user
       login password set during the installation, above.
 
-      .. figure:: /images/install-windows-screenshots/pgadmin-02initialview.png
-         :figwidth: 65%
-         :width: 60%
-         :align: center
-         :alt: Connect to the database server
-
-         Connect to the database server
-
-      .. figure::  /images/install-windows-screenshots/pgadmin-03connect.png
-         :figwidth: 65%
-         :width: 60%
-         :align: center
-         :alt: Enter password
-
-         Enter password
-
    #. Right-click on :guilabel:`Login Roles` and select :guilabel:`New Login
       Role...`
 
```

Comment:
Here add:

On the ``Properties`` tab enter the name of the OMERO database user (example: ``omero``) as the ``Role name``.  On the ``Definition`` tab, enter the password for the user twice.  Press ``OK``.

(The user name example could be omitted if covered above.)  This then follows on to the "record the name and password" bit.